.github/renovate.json

```diff
@@ -9,9 +9,17 @@
     "/^stable\\/8\\..*/"
   ],
   "dependencyDashboard": true,
-  "enabledManagers": [
-    "maven", "dockerfile", "gomod", "github-actions"
+  "prConcurrentLimit": 30,
```

Comment:
:thought_balloon: I can see this being possibly too low once we have all teams in the mono repo

Reply:
Also: are the PRs distributed equally across the projects (zeebe/operate frontend/operate backend)?

Reply:
You both mention a valid problem and indeed the Renovate behavior here is to open 30 PRs (which may be too low - on the other hand we should really strive to keep things up2date, e.g. with auto-merges) regardless of the project - since Renovate doesn't know about projects.

There is no easy way to configure this sort of fair split so again we may have to increase the limit in the future and ensure quick merges of most of the PRs - this is also in line with best practices worked on as part of https://github.com/camunda/engineering-career-board/issues/9